### PR TITLE
Add a log message when an invalid semver is encountered

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

**Comment by qixiang** [comment]
:+1:

File: freshmaker/pyxis.py [hunk 1]
```diff
@@ -169,19 +169,24 @@ def get_latest_bundles(self, index_images):
             request_params['filter'] = \
                 f'latest_in_channel==true and source_index_container_path=={path}'
 
-            def _isvalid(version):
+            def _isvalid(version, csv_name):
```
Comment (apaplaus) [inline freshmaker/pyxis.py:172]: so the `csv_name` parameter needed only for printing an error message in case of exception. Seems kind of weird.

Comment (mprahl) [reply freshmaker/pyxis.py:172]: @apaplaus it's a small helper function, so I wouldn't be too considered if it seems a little weird. That being said, I could pass in the whole bundle instead.

**Comment by qixiang** [comment]
I don't have strong opinion with adding `csv_name` to `_is_valid`, so I just give my +1 and leave the decision to Andrei :)

File: freshmaker/pyxis.py (hunk 1 above)
Comment (apaplaus) [reply freshmaker/pyxis.py:172]: Ok, if it isn't a problem for you we can leave it as it is.

**Comment by mprahl** [comment]
Can this be merged now?

**Comment by apaplaus** [comment]
Sure, I will merge it.

**Pull request merged**